src/main/java/frc/robot/RobotContainer.java

```diff
@@ -49,6 +51,15 @@ public class RobotContainer {
 
     private RobotMode robotMode = RobotMode.CORAL_LEVEL_1;
 
+    static final List<Pose2d> redBranchPoses = List.of(Constants.RED_BRANCH_A, Constants.RED_BRANCH_B, Constants.RED_BRANCH_C,
+            Constants.RED_BRANCH_D, Constants.RED_BRANCH_E, Constants.RED_BRANCH_F, Constants.RED_BRANCH_G,
+            Constants.RED_BRANCH_H, Constants.RED_BRANCH_I, Constants.RED_BRANCH_J, Constants.RED_BRANCH_K,
+            Constants.RED_BRANCH_L);
+    static final List<Pose2d> blueBranchPoses = List.of(Constants.BLUE_BRANCH_A, Constants.BLUE_BRANCH_B,
+            Constants.BLUE_BRANCH_C, Constants.BLUE_BRANCH_D, Constants.BLUE_BRANCH_E, Constants.BLUE_BRANCH_F,
+            Constants.BLUE_BRANCH_G, Constants.BLUE_BRANCH_H, Constants.BLUE_BRANCH_I, Constants.BLUE_BRANCH_J,
+            Constants.BLUE_BRANCH_K, Constants.BLUE_BRANCH_L);
```

Comment:
This should also be moved into `Constants`. It's directly related to all the constant's you're pulling in for the branch positions.